services/frontend-v3/src/components/careerInterests/CareerInterestsView.jsx

```diff
@@ -11,15 +11,18 @@ const CareerInterestsView = ({ info, relocationLocationsInfo }) => {
         dataSource={dataSource}
         renderItem={(item) => (
           <List.Item>
-            <List.Item.Meta title={item.title} description={item.description} />
+            <List.Item.Meta
+              title={item.title ? item.title : null}
+              description={item.description ? item.description : null}
```

Comment:
if the prop is not defined, we should pass undefined instead of null. So returning undefined here comes back to the same code as before. Here's one reason why https://stackoverflow.com/questions/34809178/react-js-default-prop-is-not-used-with-null-is-passed. Unless there's a reason why you are passing null here